extensions/amp-story/0.1/amp-story.js

```diff
@@ -795,10 +822,31 @@ export class AmpStory extends AMP.BaseElement {
         this.updateBackground_(this.activePage_.element);
       }
     } else {
+      this.vsync_.run({
+        measure: state => {
+          const {offsetWidth, offsetHeight} = this.element;
+          state.isLandscape = offsetWidth > offsetHeight;
+        },
+        mutate: state => {
+          this.toggleLandscapeOverlay_(state.isLandscape);
+        },
+      }, {});
       this.element.removeAttribute('desktop');
```

Comment:
Sorry I missed this, but `this.element.removeAttribute('desktop')` should also be inside the `mutate` callback.

Reply:
Done